package.json

```diff
@@ -33,7 +33,7 @@
     "contributors:gen": "all-contributors generate",
     "prestart": "yarn && yarn bootstrap",
     "boot": "yarn prestart && yarn build",
-    "start": "yarn build && yarn storybook",
+    "start": "lerna run --scope=@chakra-ui/babel-plugin build && yarn storybook",
```

Comment:
We don't have this documented anywhere but we've been trying to move towards using `manypkg` for handling most things. Could use the same script as `prebuild` in place of `lerna run`, but it's not a priority so I'll leave it up to you if you'd like to change it. 

The rest of this is great! Thanks @Zaynex!

Reply:
Done.